src/routes/JobViewRoute.js

```diff
@@ -8,104 +8,107 @@ import { ConfirmationModal } from '@folio/stripes/components';
 
-    let deleteMessageId = 'ui-local-kb-admin.job.delete.message';
-    let deleteHeadingId = 'ui-local-kb-admin.job.delete.heading';
+  let deleteMessageId = 'ui-local-kb-admin.job.delete.message';
+  let deleteHeadingId = 'ui-local-kb-admin.job.delete.heading';
 
-    if (jobClass !== '') {
-      deleteMessageId = `${deleteMessageId}.${jobClass}`;
-      deleteHeadingId = `${deleteHeadingId}.${jobClass}`;
-    }
+  if (jobClass !== '') {
+    deleteMessageId = `${deleteMessageId}.${jobClass}`;
+    deleteHeadingId = `${deleteHeadingId}.${jobClass}`;
+  }
 
-    return (
-      <>
-        <JobInfo
-          data={{
-            job
-          }}
-          isLoading={resources?.job?.isPending ?? true}
-          onClose={this.handleClose}
-          onDelete={this.showDeleteConfirmationModal}
+  return (
+    <>
+      <JobInfo
+        data={{
+          job,
+        }}
+        isLoading={resources?.job?.isPending ?? true}
+        onClose={handleClose}
+        onDelete={showDeleteConfirmationModal}
+      />
+      {showConfirmDelete && (
+        <ConfirmationModal
+          buttonStyle="danger"
+          confirmLabel={<FormattedMessage id="ui-local-kb-admin.job.delete.confirmLabel" />}
+          heading={<FormattedMessage id={deleteHeadingId} />}
+          id="delete-job-confirmation"
+          message={<SafeHTMLMessage id={deleteMessageId} values={{ name }} />}
+          onCancel={hideDeleteConfirmationModal}
+          onConfirm={handleDelete}
+          open
         />
-        {this.state.showConfirmDelete && (
-          <ConfirmationModal
-            buttonStyle="danger"
-            confirmLabel={<FormattedMessage id="ui-local-kb-admin.job.delete.confirmLabel" />}
-            heading={<FormattedMessage id={deleteHeadingId} />}
-            id="delete-job-confirmation"
-            message={<SafeHTMLMessage id={deleteMessageId} values={{ name }} />}
-            onCancel={this.hideDeleteConfirmationModal}
-            onConfirm={this.handleDelete}
-            open
-          />
-        )}
-      </>
-    );
-  }
-}
+      )}
+    </>
+  );
+};
+
+JobViewRoute.manifest = Object.freeze({
+  job: {
+    type: 'okapi',
+    path: 'erm/jobs/:{id}',
+    shouldRefresh: () => false,
+  },
+});
+
+JobViewRoute.propTypes = {
+  history: PropTypes.shape({
+    push: PropTypes.func.isRequired,
+    replace: PropTypes.func.isRequired,
+  }).isRequired,
+  location: PropTypes.shape({
+    pathname: PropTypes.string.isRequired,
+    search: PropTypes.string.isRequired,
+  }).isRequired,
+  logExportLoading: PropTypes.shape({
+    id: PropTypes.string,
+    type: PropTypes.string,
+    isLoading: PropTypes.bool
+  }),
+  mutator: PropTypes.shape({
+    job: PropTypes.object,
+  }).isRequired,
+  resources: PropTypes.shape({
+    job: PropTypes.object,
+  }).isRequired,
+  stripes: PropTypes.shape({
+    okapi: PropTypes.object.isRequired,
+  }).isRequired,
+};
 
 export default stripesConnect(JobViewRoute);
```

Comment:
You could just use `useStripes` hook instead now that this is a fn component

Reply:
Hmm, I actually don't see anywhere we use the stripes object anyway. We still need stripes connect for the manifest for the time being, but I could remove these from PropTypes?